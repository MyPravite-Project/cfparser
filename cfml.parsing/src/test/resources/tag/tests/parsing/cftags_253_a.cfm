<cfcomponent>
  <cffunction name="foo">
    <cfscript>
      removeTagList = '';
    </cfscript>
  </cffunction>
</cfcomponent>