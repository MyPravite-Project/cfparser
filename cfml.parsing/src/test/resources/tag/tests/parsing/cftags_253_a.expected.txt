START:cfcomponent
  =========TAG=================
  <cfcomponent>
  <cffunction name="foo">
    <cfscript>
      removeTagList = '';
    </cfscript>
  </cffunction>
</cfcomponent>
  START:cffunction
    =========TAG=================
    <cffunction name="foo">
    <cfscript>
      removeTagList = '';
    </cfscript>
  </cffunction>
    START:cfscript
      =========TAG=================
      <cfscript>
      removeTagList = '';
    </cfscript>
      !!!removeTagList = ''
    END:cfscript
  END:cffunction
END:cfcomponent
